supervisor.conf.j2

```diff
@@ -6,3 +6,8 @@ environment=DJANGO_SETTINGS_MODULE="{{ project_name }}.settings.{{ django_settin
 
 [group:{{ project_name }}]
 programs=gunicorn-{{ project_name }}
```

Comment:
Ah, @rascalking is correct.  You need to add the huey program to the project group.  Not clear from your comment if you're getting the above errors when you do that and deploy.